lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -373,7 +382,7 @@ private EvidenceRequest getEvidenceRequestForF2F(List<VerifiableCredential> vcs)
             return null;
         }
 
-        return new EvidenceRequest(minViableStrengthOpt.getAsInt());
+        return new EvidenceRequest(null, minViableStrengthOpt.getAsInt());
```

Comment:
Should be covered by regression tests but probably worth verifying these changes don't change the behaviour/format of what we're sending to F2F in the evidence claim

Reply:
Can see there are a few scenarios for this- https://github.com/govuk-one-login/ipv-core-tests/blob/d995435539503fdc926456034951856a578f5fcc/src/test/resources/features/Face2Face.feature#L251